.github/workflows/label.yml

```diff
@@ -6,7 +6,13 @@
 # https://github.com/actions/labeler/blob/master/README.md
 
 name: Labeler
-on: [pull_request]
+on:
+  pull_request_target:
+    types:
+      - opened        # when PR is opened
+      - edited        # when PR is edited
+      - synchronize   # when code is added
+      - reopened      # when a closed PR is reopened
```

Comment:
Only “opened” , PLEASE. So painful having it keep improperly label PRs after you’ve already fixed it once. The labeler is completely worthless anyway. But if it must exist, please only on opened.

Reply:
done, I would like to try and if it does not work rollback